bin/cylc-insert

```diff
@@ -31,6 +31,17 @@ will not be added to the pool if it catches up to another task with the
 same ID).
 
 See also 'cylc submit', for running tasks without the scheduler.
+
+TASK_GLOB is a pattern to match task proxies or task families,
+or groups of them (note in this case you cannot match against cycle points):
+* TASK-NAME-GLOB[:TASK-STATE]
+* FAMILY-NAME-GLOB[:TASK-STATE]
+
+For example, to match:
+* all tasks in the submitted status: ':submitted'
+* retrying tasks in 'BAR' family: '*/BAR:retrying' or 'BAR.*:retrying'
+* retrying tasks in 'BAR' or 'BAZ' families: '*/BA[RZ]:retrying' or
+  'BA[RZ].*:retrying'
 """
```

Comment:
This isn't correct actually @sadielbartholomew -

The `cylc insert` command inserts new `waiting` (or `runahead`) task proxies into the task pool at a specific given cycle point.  So on the command line:
-  `[:TASK-STATE]` is ignored (and should be disallowed)
-  a specific cycle point is required

i.e. the second arg has to be one of these forms:
```
CYCLE-POINT/TASK-NAME-GLOB
CYCLE-POINT/FAMILY-NAME-GLOB
TASK-NAME-GLOB.CYCLE-POINT
FAMILY-NAME-GLOB.CYCLE-POINT
```

Reply:
Oh dear, thanks for the correction Hilary! I should have manually checked what works as globs for ``insert`` rather than assuming what is valid based on comments on the Issue etc.  :grimacing:. I'll update this shortly.